CONTRIBUTING.md

```diff
@@ -1,29 +1,5 @@
 # How to contribute
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
-
-## Contributor License Agreement
-
-Contributions to this project must be accompanied by a Contributor License
-Agreement. You (or your employer) retain the copyright to your contribution,
-this simply gives us permission to use and redistribute your contributions as
-part of the project. Head over to <https://cla.developers.google.com/> to see
-your current agreements on file or to sign a new one.
-
-You generally only need to submit a CLA once, so if you've already submitted one
-(even if it was for a different project), you probably don't need to do it
-again.
-
-### Workflow
-
-* All PRs must be pulled from [forks](./DEVELOPMENT.md#checkout-your-fork)
-* All PRs must [be reviewed](#code-reviews)
-
-## Code reviews
-
-All submissions, including submissions by project members, require review. We
-use GitHub pull requests for this purpose. Consult [GitHub Help] for more
-information on using pull requests.
-
-[GitHub Help]: https://help.github.com/articles/about-pull-requests/
+Please see the Knative Community
```

Comment:
Can you use the text from https://github.com/knative/serving/blob/master/CONTRIBUTING.md for consistency? But instead of `Knative Serving's overall contribution guidelines` say `Knative's overall contribution guidelines`.